internal/node/config.go

```diff
@@ -49,6 +48,10 @@ type NodeConfiguration struct {
 func (c *NodeConfiguration) Validate() bool {
 	c.Errors = make([]error, 0)
 
+	if c.MachinePoolSize < 1 {
```

Comment:
instead of erroring here, can we warn and set it to 1?

Reply:
Not ideal IMHO, means you can have a malformed config and get away with it.

Reply:
Error gives the user a chance to fix their faulty config..

Reply:
meh, okay